### PR TITLE
Modified djstation and added random intercom

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Made the ruin look more like a ruin and changed the intercom to remove master key.

## Why / Balance
Direction wanted the master key intercom gone, and we got an issue to make the ruins look more dirty.

## Technical details

YAML changes.
## Media
image
image

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.

**Changelog**

No cl

### CODE REVIEW

**Comment by deltanedas** [comment]
fixes some issue i made

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/intercom.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+- type: entity
+  parent: MarkerBase
+  id: RandomIntercom
+  name: Random intercom
+  components:
+  - type: Sprite
+    layers:
+      - sprite: DeltaV/Structures/Wallmounts/intercom.rsi
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/intercom.yml]: unindeng 2 spaces

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/intercom.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        state: random_intercom
+  - type: RandomSpawner
+    offset: 0
+    prototypes:
+      - IntercomSupply
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/intercom.yml]: above

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/intercom.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      - IntercomService
+      - IntercomScience
+      - IntercomMedical
+      - IntercomEngineering
+      - IntercomCommon
+      - IntercomConstructed
+      - IntercomAssembly
+    rarePrototypes:
+      - IntercomCommand
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/Random/intercom.yml]: above

File: Resources/Textures/DeltaV/Structures/Wallmounts/intercom.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from /tg/station at https://github.com/tgstation/tgstation/commit/6bfe3b2e4fcbcdac9159dc4f0327a82ddf05ba7bi. Modified by Radezolid for random intercom spawner",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+	{
+	    "name": "random_intercom",
+        "directions": 4
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Structures/Wallmounts/intercom.rsi/meta.json]: indent is sus on this line

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/DeltaV/Structures/Wallmounts/intercom.rsi/meta.json (hunk 4 above)
Comment (Radezolid) [reply Resources/Textures/DeltaV/Structures/Wallmounts/intercom.rsi/meta.json]: It was a stupid error, tab vs 4 spaces. Didn't notice it at the time. Fixed.

Comment (deltanedas) [reply Resources/Textures/DeltaV/Structures/Wallmounts/intercom.rsi/meta.json]: now indent the state {} block again :trollface:

Comment (Radezolid) [reply Resources/Textures/DeltaV/Structures/Wallmounts/intercom.rsi/meta.json]: Done

**Review by Colin-Tel — APPROVED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**